### DISCUSSION TITLE
Layout de impressão

### DISCUSSION THREAD

**Discussion by leosass** [RamonSilva20/mapos#1578] [Ideas]
Galera como personalizar o layout de impressão?
Apenas deixar o modelo mais visualmente bonito?

Uma referência é o da Operand. Se quiserem mando aqui.

Obrigado.

**Comment by Andrefiel** [comment]
Manda aí o modelo, também queria alterar o local do QRcode do PIX e incluir os dados bancários pq nem sempre o cliente usa PIX

**Comment by Pr3d4dor** [comment]
Basta você alterar o arquivo imprimir_os.php e deixar do seu gosto.

**Reply by SuperPorte** [reply]
Onde fica esse arquivo?
Queria adicionar um canhoto para destacar e entregar ao cliente .

**Reply by Pr3d4dor** [reply]
@bernarcom Fica dentro da pasta de views em: application/views/os/imprimirOs.php

**Comment by andremp15** [comment]
Oi pessoal vou deixar a minha ideia que eu tenho a respeito emissão da os de entrada essa seria a minha ideia vou postar uma imagem fiz no Corel draw
![Uploading IMG-20221230-WA0029.jpg…]()

**Comment by andremp15** [comment]
![IMG-20221230-WA0029](https://user-images.githubusercontent.com/117945999/210121717-a780093b-4a85-4568-81bc-3a7bdf811fdd.jpg)
